Resources/ServerInfo/_NF/Guidebook/Rules/6_Safezone.xml

```diff
@@ -18,4 +18,8 @@
     -- Stealing ships or ships' cargo.
     -- Selling someone else's ship you hadn't pirated outside the safe zone.
     -- Joining a crew under false pretenses with the intent to mutiny and take the ship.
+    - Players must make best efforts to keep the docks at Frontier Outpost free and clear for the use of others.
+    -- The Trade Outpost / Trade Mall can be used if you wish to remain docked to a station to conduct business.
```

Comment:
```suggestion
    -- Docking extensions may be granted for the Trade Outpost / Trade Mall at the discretion of the SR and STC.
```

Reply:
also, remove the bit under section 13 for "granting docking extensions"

Reply:
Honestly, this would probably be best handled in a separate docking section.

Reply:
I don't think we have enough docking rules to really warrant an entire extra rule section on it. I think it's fine as it is under the safezone rules.

Reply:
Additionally, docking rules enforcement is optional on the trade stations. Most of the time it isn't.